R/snapshot.R

```diff
@@ -704,7 +704,11 @@ renv_snapshot_description <- function(path = NULL, package = NULL) {
 renv_snapshot_description_impl <- function(dcf, path = NULL) {
 
   # infer remotes for packages installed from sources
-  dcf <- renv_snapshot_description_infer(dcf)
+  #
+  # TODO: This causes problems with status(), and is potentially
+  # far too expensive if we have a number of packages installed
+  # from sources.
+  # dcf <- renv_snapshot_description_infer(dcf)
```

Comment:
File issue to look into?

Reply:
https://github.com/rstudio/renv/issues/1527